src/main/java/edu/sc/seis/launch4j/CreateLaunch4jXMLTask.java

```diff
@@ -21,12 +21,12 @@
 public class CreateLaunch4jXMLTask extends DefaultTask {
     @OutputFile
     public File getXmlOutFile() {
-        return ((Launch4jPluginExtension) getProject().getExtensions().getByName(Launch4jPlugin.LAUNCH4J_CONFIGURATION_NAME)).getXmlOutFileForProject(getProject());
+        return ((Launch4jPluginExtension) getExtensions().getByName(Launch4jPlugin.LAUNCH4J_CONFIGURATION_NAME)).getXmlOutFileForProject(getProject());
```

Comment:
memo: I need to check this change doesn't change behavior.

Reply:
I think this breaks the behavior.
if we use `getProject().getExtensions()`, we will get extensions of a project but extensions of task with `this.getExtensions()`.
In most case, this *may* not change the behavior (not tested) but if some added extensions to the task(s), behavior can be changed.